aiohttp/helpers.py

```diff
@@ -747,11 +747,9 @@ def charset(self) -> Optional[str]:
     def content_length(self) -> Optional[int]:
         """The value of Content-Length HTTP header."""
         content_length = self._headers.get(hdrs.CONTENT_LENGTH)
-
-        if content_length is not None:
-            return int(content_length)
-        else:
+        if content_length is None:
             return None
+        return int(content_length)
```

Comment:
One-liner?
```suggestion
        return None if content_length is None else int(content_length)
```

Reply:
as long as it doesn't wrap, sound good to me